src/errors.ts

```diff
@@ -29,20 +29,6 @@ export class ReadOnlyError extends AzFuncTypeError {
 
 export function ensureErrorType(err: unknown): Error & Partial<AzFuncError> {
```

Comment:
How about we change the return type of `ensureErrorType` to something like this. That way we should get a build error if we forget and try to set the message directly

```
export interface TodoError extends Error, Partial<AzFuncError> {
    /**
     * Use `setErrorMessage` to set the error message
     */
    readonly message: string;
}

export function ensureErrorType(err: unknown): TodoError {
```

Reply:
Added the interface